### DISCUSSION TITLE
Widget with Clock, just Alarms on Samsung

### DISCUSSION THREAD

**Discussion by Lauch3000** [breezy-weather/breezy-weather#953] [Q&A] [answered]
On a Samsung Phone if I click on the Time on a Breezy Weather Widget with a clock it starts the samsung clock app, but just the alarm part. And not the whole "clock" app with timers, world clock, etc. Can I change this somewhere? Or is it planned to do this customiziable?

regards, Lauch3000

**Comment by papjul** [comment, marked as answer]
Seriously? The question was just asked 38 minutes ago

Duplicate of #648 and #952

Please report the issue to Samsung, or they will never fix it!

(The issue being: other tabs are not available)

**Comment by Lauch3000** [comment]
Oh, ok. Sorry for that. I didn´t do a search on closed discussions. Ok, than I will try my luck on samsung.

**Reply by papjul** [reply]
Don't worry, it's more at Samsung that I'm upset right now. If I get 3 bug reports, it means there are actually way more users with the same issue waiting silently to get it fixed

**Reply by Lauch3000** [reply]
I know what you mean. And yes, I guess there are more Users. But it is an old issue, dont know if it get fixed. I had it in different Apps with clock widgets. 

In some (weather) Apps it works, for example on weawow or wetter.com (german). So I thought it is somethink like a wrong link/target which is called. 

Ok, I will see if I get some Infos from Samsung for this.

**Reply by papjul** [reply]
> In some (weather) Apps it works

Ah? No one mentioned that in other issues

If you can find an open source app on which it works, I can take a look

**Reply by Lauch3000** [reply]
Ok, I will try to find a app with a clock widget. I know Weawow works https://play.google.com/store/apps/details?id=com.weawow&hl=de&gl=US but is not OS :(
I search for some, maybe I will find any.

**Reply by papjul** [reply]
Weawow has a dirty hack when manufacturer is Samsung. They have a hardcoded list of clock apps, and they loop through all of them, asking the package manager if the package exists to open it instead. I will not implement that something like that, sorry, please ask Samsung to fix their bug (remember you paid for your phone and for this clock app, so you have a right to support). Otherwise, just switch to a different clock app ;)

**Reply by papjul** [reply]
What you need to tell Samsung is that when you open the clock app through the `AlarmClock.ACTION_SHOW_ALARMS` intent, the clock app opens but other tabs are not available. This issue does not reproduce in other clock apps.

**Reply by Lauch3000** [reply]
Ok, this is really ugly from weawow. And I understand, that you wont implement such a hack. It is also a hydra, you add one app and another is coming... 

Thanks for the technical Details. I'm a developer too, but I don't have any clue about android Development. So, I will annoy Samsung with this :)
